resolwe/permissions/tests/test_shortcuts.py

```diff
@@ -159,21 +159,27 @@ def test_user_permissions(self):
             {'permissions': ['edit', 'view'], 'type': 'group', 'id': self.group1.pk, 'name': 'Test group 1'},
         ]
         perms = get_object_perms(self.collection, self.user1)
-        six.assertCountEqual(self, expected_perms, perms)
+
+        def _sort_perms(perms):
```

Comment:
Define this as a class method and also fix permissions assertions in above test.